tests/test_sqldiff.py

```diff
@@ -36,6 +40,42 @@ def test_sql_diff_without_proxy_models(self):
     def test_sql_diff_with_proxy_models(self):
         self._include_proxy_models_testing(True)
 
+    def test_format_field_names(self):
+        instance = MySQLDiff(
+            apps.get_models(include_auto_created=True),
+            vars(self.options),
+            stdout=self.tmp_out,
+            stderr=self.tmp_err,
+        )
+        expected_field_name = ['name', 'email', 'address']
+        self.assertEquals(instance.format_field_names(['Name', 'EMAIL', 'aDDress']), expected_field_name)
+
+    def test_sql_to_dict(self):
```

Comment:
I would like to split these up in 3 tests for each database and have the `@pytest.skipif(...)` decorator on them to test only when that database engine is used.

This because even though the test right now works, it does not really test what you would expect. In most of our test runs no matter which Diff engine is used it will always run on `sqlite` since that it the database engine for most test runs.

Also I would love to test the failure case as well. That way we can actually see that this PR is useful and it's easier to test regressions or find to find out that database engine change their behaviors.

So I think this test should be split in 3, one for each database server and it should monkeypatch and `assertEquals` the failure case in MySQL as well.